events/elections/2021/README.md

```diff
@@ -190,7 +235,8 @@ the [Steering Committee Election Charter]
 [CIVS]: http://civs.cs.cornell.edu/
 [IRV method]: https://www.daneckam.com/?p=374
 
-[2021 candidate bios]: https://github.com/kubernetes/community/tree/master/events/elections/2021
+[`hack/verify-steering-election.sh`]: https://git.k8s.io/community/hack/verify-steering-election.sh
+[2020 candidate bios]: https://github.com/kubernetes/community/tree/master/events/elections/2021
```

Comment:
2021

Reply:
```suggestion
[2020 candidate bios]: https://github.com/kubernetes/community/tree/master/events/elections/2020
```
This is a ref link above pointing to the 2020 bios as examples:
https://github.com/kubernetes/community/pull/5326/files#diff-b2bd5da91ac54889f08d53c8a107d9097af1a7a1d0289d149d9e866a2cbcab81R148-R150